src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+
+New Relic provides many ways to improve visibility. They let you monitor, troubleshoot, and improve digital performance. Here's a brief overview of our most popular capabilities and features.
+
+
+# Alerts [#alerts]
+<SideBySide>
+  <Side>
+Alerts help you identify issues that look unusual in your applications and code. New Relic includes some <InlinePopover type="alerts" /> when you install it, and you can also create your alerts. You receive alerts through tools like PagerDuty, ServiceNow, Jira, Slack, and more. Let us help you know what's critical, manage the suppressed noise, and mitigate alert fatigue. 
+  
+Learn more about alerts [here](/docs/alerts-applied-intelligence/overview/).
+
+  </Side>
+  <Side>
+    <img
+      title="New Relic alerts"
+      alt="New Relic alerts"
+      src={NRAlertsMain}
+    />
+  </Side>
+  </SideBySide>
+
+# APM [#apm]
+<SideBySide>
+  <Side>
+
+  <img
+    title="New Relic APM"
+    alt="New Relic APM"
+    src={NRApmMain}
+  />
+
+  </Side>
+  <Side>
+
+  New Relic instruments your application by using one of our many language agents. These agents collect metrics from your application and then send the metrics to our <InlinePopover type="apm" />. It lets you monitor your application using pre-built dashboards. 
+  
+  Learn more about APM [here](/docs/apm/new-relic-apm/getting-started/introduction-apm/).
+
+  </Side>
+  </SideBySide>
+
+# Browser [#browser]
+<SideBySide>
+  <Side>
+
+  The <InlinePopover type="browser" /> capability collects data from the navigation timings in all end users' browsers. It sends that data directly to New Relic while blending with the service that served the requests. When you load that page URL, you can easily see which back-end transaction fulfilled that request. 
+
+  Learn more about browser [here](/docs/browser/browser-monitoring/getting-started/introduction-browser-monitoring/).
+
+  </Side>
+  <Side>
+
+<img
+  title="New Relic browser"
+  alt="New Relic browser"
+  src={NRBrowserMain}
+/>
+
+  </Side>
+  </SideBySide>
+
+# Dashboards [#dashboards]
+<SideBySide>
+  <Side>
+
+  <img
+  title="New Relic dashboards"
+  alt="New Relic dashboards"
+  src={NRDashboardsMain}
+/>
+
+  </Side>
+  <Side>
+
+  Our UI allows you to create custom charts and share them easily. Our <InlinePopover type="dashboards" /> capability lets you collect and visualize your data preferences across our platform. Tailor your charts to display the information you need. Access and analyze your data from any location within our platform.
+
+  NRQL, short for New Relic query language, is a powerful SQL-like tool for accessing detailed data from New Relic. It lets you gather specific info about your applications, hosts, and business operations. By using NRQL, you can gain valuable insights into the performance and behavior of your environment. 
```

Comment:
```suggestion
  New Relic Query Language (NRQL) is a powerful SQL-like tool for accessing detailed data from New Relic. It lets you search for specific information about your applications, hosts, and business operations. Using NRQL empowers you to gain valuable insights into the performance and behavior of your system. 
```

Reply:
NRQL is also used on its own, too... If we _have_ to keep NRQL in dashboards, then I'd introduce NRQL e arlier on and weave it into the intro text, rather than having it be an abrupt two part paragraph

Reply:
Agreed and good callout, we have a whole section on NRQL in the docs so it makes sense to call it out in some capacity here as well.

Reply:
Hey @paperclypse, do you agree with having a new section for NRQL?

Reply:
It is a powerful and useful tool that New Relic provides and it's what so many of our features and capabilities are built on. I'm OK with adding a new NRQL section. (I also think it's fine to add it as a fast follow, too. I don't see the need for it to be a blocker if it feels like it'd take a lot of time to do it right.) @nbaenam

Reply:
Done!